source/slang/slang-ir-diff-jvp.cpp

```diff
@@ -3,10 +3,303 @@
 
 #include "slang-ir.h"
 #include "slang-ir-insts.h"
+#include "slang-ir-clone.h"
 
 namespace Slang
 {
 
+struct JVPTranscriber
+{
+
+    // Stores the mapping of arbitrary 'R-value' instructions to instructions that represent
+    // their differential values.
+    Dictionary<IRInst*, IRInst*> instMapD;
```

Comment:
Naming: we tend to name dictionaries that conveys its intention more clearly: e.g. mapInstToDifferentiatedValue would be proper here.